manifests/0000_90_kube-apiserver-operator_04_servicemonitor-apiserver.yaml

```diff
@@ -106,6 +106,32 @@ metadata:
     exclude.release.openshift.io/internal-openshift-hosted: "true"
 spec:
   groups:
+  - name: aggregated-apis
+    rules:
+    - alert: AggregatedAPIErrors
+      annotations:
+        message: "An aggregated API {{ $labels.name }}/{{ $labels.namespace }} have reported errors."
+        description: "The number of error ({{ $labels.reason }}) has increased for an aggregated API {{ $labels.name }}/{{ $labels.namespace }} in the last 2 minutes."
+      expr: |
+        sum by(name, namespace, reason)(increase(aggregator_unavailable_apiservice_count[2m])) > 1
+      labels:
+        severity: warning
+    - alert: AggregatedAPIDown
+      annotations:
+        message: "An aggregated API {{ $labels.name }}/{{ $labels.namespace }} down."
```

Comment:
At the moment this metric doesn't work. I suspect that the upstream code is broken. In an `HA` setup one instance can set its instance of the metric to `1` whereas the other instance might flip its own instance to `0` preventing the first instance from changing it to `0`(I need to investigate it). But definitely something fishy is going on because I saw some instances reported a service as unavailable whereas it was available.

@sttts FYI

Reply:
suppose that we fix it upstream, will this alert be present on downgrade?